app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -3024,6 +3050,19 @@ class BrowserTabViewModel @Inject constructor(
             }
 
             "screenUnlock" -> screenUnlock()
+
+            "getUserValues" -> if (id != null) {
```

Comment:
I know the name doesn't come from our stuff but the name of these is pretty bad. Gives 0 context about what feature this is about. I think we should probably separate by featureName rather than method and the delegate to whoever owns the feature.